### PR TITLE
refactor: reduce bundle size

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/chart/babel.config.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        '@babel/preset-env',
+        env === 'polyfill'
+          ? {
+              useBuiltIns: 'usage',
+              debug: false,
+              modules: 'auto',
+              corejs: 3,
+            }
+          : {},
+      ],
```
Comment (shiren) [inline apps/chart/babel.config.js:16]: 아 이렇게 했구나. 저는 좋아요

**Review by shiren — APPROVED** [review]
LGTM

**Review by lja1018 — APPROVED** [review]
리뷰완료
고생하셨습니다!

**Review by jungeun-cho — APPROVED** [review]
1) npm 배포 결과물에서 index.html은 내보내지 않아야합니다.
웹팩 설정에서 `html-webpack-plugin` 지워도될것같아요.

2) color.ts 파일에 @license 명시해준 부분이 있더라고요. 
이부분 명시가 꼭 필요한가해서요. 


스크린샷 2021-01-21 00 31 57

코드 상에 표시만해도되면 따로 npm 배포에 포함되지 않아도 될 것 같아요. 
`terser-webpack-plugin`에서 `extractComments: false`옵션을 주는건어떻할까요?
https://webpack.js.org/plugins/terser-webpack-plugin/#extractcomments
```js
new TerserPlugin({
  extractComments: false,
```

**Pull request merged**